README.md

```diff
@@ -1 +1,5 @@
-# code-review-trial-1
\ No newline at end of file
+# code-review-trial-1
+
+### Technologies
+- HTML
+- CSS
```

Comment:
Write a descriptive readme file

Reply:
- [ ] I think it would be better to give a polite suggestion. Some students feel depressed by the work, so a polite suggestion will help them overcome the stress